src/web/app/pages-instructor/instructor-session-base-page.component.ts

```diff
@@ -470,6 +511,40 @@ export abstract class InstructorSessionBasePageComponent {
     modal.componentInstance.requestId = resp.error.requestId;
     modal.componentInstance.errorMessage = resp.error.message;
   }
+
+  triggerModelChange(data: SessionEditFormModel): void {
+    const { submissionStartDate, submissionEndDate, submissionStartTime, submissionEndTime } = data;
+
+    const startDate = new Date(submissionStartDate.year, submissionStartDate.month, submissionStartDate.day);
+    const endDate = new Date(submissionEndDate.year, submissionEndDate.month, submissionEndDate.day);
+
+    if (startDate > endDate) {
+      this.sessionEditFormModel = {
+        ...data,
+        submissionEndDate: submissionStartDate,
+        submissionEndTime: submissionStartTime.hour > submissionEndTime.hour ? submissionStartTime : submissionEndTime,
```

Comment:
@Zxun2 you'll have to change to logic here as well! Sorry for not pointing it out earlier (: You can check by setting:
- Opening to 2 Jul 2024, 2359
- Closing to 3 Jul 2024, 2300
- Change opening to 4 Jul 2024 (2359)
- Closing changes to 4 Jul 2024 (2300)
<img width="796" alt="Screenshot 2023-07-12 at 1 08 17 AM" src="https://github.com/TEAMMATES/teammates/assets/48304907/f63d3f14-7691-47ce-8ada-f8efafd94d8b">

Reply:
Thanks for the spot😅